src/Stratis.Bitcoin.Features.RPC/FundRawTransactionResponse.cs

```diff
@@ -1,17 +1,23 @@
 ﻿using NBitcoin;
+using Newtonsoft.Json;
 
 namespace Stratis.Bitcoin.Features.RPC
 {
     public class FundRawTransactionResponse
     {
+        [JsonProperty(PropertyName = "hex")]
```

Comment:
Should this be `hex` ?

Reply:
Yes; the actual field returned by bitcoind is named `hex`. I didn't want to have to make unnecessary changes to NBitcoin, so the name stayed.